client/jetpack-cloud/sections/agency-dashboard/sites-overview/site-error-content.tsx

```diff
@@ -12,11 +12,14 @@ export default function SiteErrorContent( { siteUrl }: { siteUrl: string } ) {
 	};
 
 	return (
-		<div className="sites-overview__error-container">
+		<div className="sites-overview__error-container" data-testid="site-error-container">
 			<span className="sites-overview__error-icon">
 				<Gridicon size={ 18 } icon="notice-outline" />
 			</span>
-			<span className="sites-overview__error-message sites-overview__error-message-large-screen">
+			<span
+				className="sites-overview__error-message sites-overview__error-message-large-screen"
+				data-testid="site-error-message"
```

Comment:
How about we get rid of the `data-testid` and use `screen.getByText` instead to get the element since it is close to what the user sees on the screen?

Reply:
@yashwin I pushed the suggested changes in the unit tests.